.flowconfig

```diff
@@ -18,4 +18,18 @@
 [options]
 module.system.node.resolve_dirname=node_modules
 
+module.file_ext=.js
+module.file_ext=.jsx
+module.file_ext=.json
+module.file_ext=.native.js
```

Comment:
does that mean .spec.js is not included? Intentional?

Reply:
I don't think so, no. Should be included anyway since `.js` is included.

Reply:
yes. see https://momentjs.com/docs/#/parsing/string/

Reply:
then why did you add .native.js separately?

Reply:
are there even any `.native.js` files in the webapp?

Reply:
Probably not :D Copied from the native flowconfig, will revert.